### PR TITLE
SAK-48653 Rubrics: Remove BS3 classes in favor of BS5

### PR DESCRIPTION
**Pull request by jkozar2**
Contribution to the ongoing Bootstrap 3 to 5 project: https://docs.google.com/document/d/1HCNLBswh5guaPLAEPkKnuadHBy8HKh_q3D5q-cAeUA8
I replaced a number of classes as specified in the document, and found a form that does validation and "successful save" notification based on Bootstrap 3 classes. I replaced them with custom styling and JQuery methods.

### CODE REVIEW

File: webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric-criteria.js [hunk 1]
```diff
@@ -219,7 +219,7 @@ export class SakaiRubricCriteria extends RubricsElement {
             </div>
           </div>
           <div class="banner-container">
-            <div class="sak-banner-success d-none save-success has-success fade">
+            <div class="sak-banner-success" style="display:none">
```
Comment (kunaljaykam) [inline webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric-criteria.js:222]: Was it necessary to remove the class `d-none`?

**Review by adrianfish — CHANGES_REQUESTED** [review]

**Comment by ern** [comment]
Joe can you add the SAK ticket for this in title

File: webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric.js [hunk 2]
```diff
@@ -288,21 +288,14 @@ export class SakaiRubric extends RubricsElement {
         if (r.ok) {
 
           if (saveSuccessLbl) {
-            saveSuccessLbl.classList.remove('d-none');
-            saveSuccessLbl.classList.add('in');
+            $(saveSuccessLbl).show();
```
[the remark this answers is not in the document]
Comment (jkozar2) [reply webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric.js]: I had done this to replicate the fade-out behavior that the existing Bootstrap stuff was doing. JQuery was the best other option I could come up with. I then considered trying to do it manually somehow, but on my latest change I just got rid of the fade and have it disappearing instantly after 5 seconds. So if we're ok without the fade this is taken care of!

**Comment by jkozar2** [comment]
I couldn't find a SAK ticket number to add to this, either a master ticket for the project or a Rubrics one. Should I make one, or is there a proper ticket for this I'm just not finding?

**Comment by jkozar2** [comment]
Made this ticket for tracking the issue:
https://sakaiproject.atlassian.net/browse/SAK-48653

**Review by kunaljaykam — APPROVED** [review]

**Pull request merged**